docs/docs/getting-started.server.mdx

```diff
@@ -137,6 +139,7 @@ For more info on the aforementioned tools, please see their dedicated sections:
 [¶ Svelte][svelte],
 [¶ Theme UI][themeui], or
 [¶ Vue][vue].
+[¶ Solid][solid],
```

Comment:
Move this above Svelte. The list is sorted alphabetically.

Reply:
thanks.

Reply:
Not sure if you’re still working on this or thought it was fixed. But to be as clear as possible: I meant: this line L144, should be above L141 (Svelte)! :)

Reply:
got it!